src/test/java/regres/get/users/single/user/GetSingleUserNegativeDataDrivenTest.java

```diff
@@ -17,12 +17,12 @@
 
 @RunWith(SerenityParameterizedRunner.class)
 @WithTags({
-        @WithTag(type= "type", name = "GET")
+        @WithTag(type = "type", name = "GET")
 })
 public class GetSingleUserNegativeDataDrivenTest {
 
     @TestData
-    public static Collection<Object[]> data(){
+    public static Collection<Object[]> data() {
         return Arrays.asList(new Object[][]{
                 {100},
```

Comment:
Keep in mind that if you choose to use random IDs for the above tests, this may fail. In this case we can keep to your approach using predefined ids for above tests, but make sure you delete the users after tests ran